web/documentserver-example/python/src/views/actions.py

```diff
@@ -524,19 +525,34 @@ def reference(request):
             if userAddress == request.META['REMOTE_ADDR']:
                 fileName = fileKey['fileName']
 
+    if not fileName and 'link' in body:
+        if docManager.getServerUrl(False, request) not in body['link']:
+            data = {
+                'url': body["link"],
+                'directUrl': body["link"]
+            }
+            return HttpResponse(json.dumps(data), content_type='application/json')
+
+        urlObj = urlparse(body['link'])
```

Comment:
In Python, they walk like snakes. 🐍
See [PEP 8](https://peps.python.org/pep-0008/#function-and-variable-names).

Reply:
> В Python они ходят как змеи. 🐍 См. [ ПКП 8 ](https://peps.python.org/pep-0008/#function-and-variable-names) .

Changed from camelCase to snake_case.